app/templates/components/supplementary-file-browser.hbs

```diff
@@ -2,8 +2,19 @@
     width="99%" height="700"}}
 {{/file-renderer}}
 
+<div class="row">
+    <div class="col-xs-8">
+        <p class="f-w-lg p-b-md">{{selectedFile.name}}</p>
+    </div>
+    <div class="col-xs-4 pull-right text-nowrap">
```

Comment:
don't pull-right the col-xs-4, make a div inside it and pull that to the right or right align the text.